item.rb

```diff
@@ -0,0 +1,34 @@
+require_relative './authors/author'
+
+class Item
+  attr_accessor :publish_date, :id, :archived
+
+  def initialize(publish_date, archived: true)
+    @id = Random.rand(1...1000)
+    @publish_date = publish_date
+    @archived = archived
+  end
+
+  def genre=(genre)
+    @genre = genre
+    genre.music_album_items << self
+  end
+
+  def label=(label)
+    @label = label
+    label.items.push(self) unless label.items.include?(self)
+  end
+
+  def can_be_archived?
+    2022 - @publish_date > 10
+  end
```

Comment:
- [ ] Kindly make sure you update the `can_be_archived?` method to correctly match the requirement. It should return true if published_date is older than 10 years as you did. But the issue is that it won't be updated when you move the next years(2023, 2024,...). For that, I suggest you use the **Date.new** method, this will remain updated in the future.
Example:
```ruby
    now = Date.today.year
    published = @publish_date.to_i
    (now - published) >= 10
```
Or much better than this by using **year, month, and day**